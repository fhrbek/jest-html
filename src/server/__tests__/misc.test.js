const OBJ = {
  just: 'an',
  object: 'with',
  some: 'attributes',
  inIt: { foo: 3, bar: 4 },
};

const MARKDOWN = `
# Title 1

## Title 2

A paragraph with *some* **formatted** text and [a link](http://github.com/guigrpa).

* Some
* Bullets
* There you go!

\`\`\`js
// Some JavaScript
const a = {
  b: 3,
};
\`\`\`

Is everything OK?
`;

describe('miscellaneous', () => {
  it('should correctly show JSON', () => {
    expect(OBJ).toMatchSnapshot();
  });

  it('should correctly show Markdown', () => {
    expect(MARKDOWN).toMatchSnapshot();
  });
})
