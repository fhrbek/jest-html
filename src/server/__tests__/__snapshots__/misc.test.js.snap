// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`miscellaneous should correctly show JSON 1`] = `
Object {
  "inIt": Object {
    "bar": 4,
    "foo": 3,
  },
  "just": "an",
  "object": "with",
  "some": "attributes",
}
`;

exports[`miscellaneous should correctly show Markdown 1`] = `
"
# Title 1

## Title 2

A paragraph with *some* **formatted** text and [a link](http://github.com/guigrpa).

* Some
* Bullets
* There you go!

\`\`\`js
// Some JavaScript
const a = {
  b: 3,
};
\`\`\`

Is everything OK?
"
`;
